app-frontend/src/app/pages/projects/edit/annotate/annotate.controller.js

```diff
@@ -313,27 +301,61 @@ export default class AnnotateController {
                     `,
                     {closeButton: false}
                 ).on('mouseover', (e) => {
-                    if (feature.geometry.type === 'Polygon') {
-                        e.target.setStyle({'color': '#ff5b4d'});
-                    } else if (feature.geometry.type === 'Point') {
-                        e.target.setIcon(L.divIcon({'className': 'annotate-hover-marker'}));
+                    if (!this.disableSidebarAction && !this.clickedId) {
+                        this.hoveredId = feature.properties.id;
+                        this.setLayerStyle(feature, e, '#E57373', 'annotate-hover-marker');
+                        this.scrollToItem(this.hoveredId);
+                        this.$scope.$evalAsync();
                     }
                 }).on('mouseout', (e) => {
-                    if (feature.geometry.type === 'Polygon') {
-                        e.target.setStyle({'color': '#3388ff'});
-                    } else if (feature.geometry.type === 'Point') {
-                        e.target.setIcon(L.divIcon({'className': 'annotate-marker'}));
+                    if (!this.disableSidebarAction && !this.clickedId) {
+                        this.hoveredId = '';
```

Comment:
better to just `delete this.hoveredId` so we can treat the string as a boolean when determining validity